brainmap-cms/settings_custom.py

```diff
@@ -23,9 +23,20 @@
 # BRANDING
 ########################
 
+_UTHSCSA_BRANDING = [
+    "uthscsa",
+    "brainmap-cms/img/org_logos/uthscsa-logo-white.png",
+    "branding-logo--tall",
```

Comment:
__Pending Design Decision__
See https://github.com/TACC/Core-CMS/pull/414#user-content-fnref-2-b6a6fd48ee3671fa2238e2a7946fe9ba (might need to scroll up a bit on that page).

Reply:
Going with tall one.